ocs_ci/helpers/dr_helpers.py

```diff
@@ -1083,3 +1084,117 @@ def verify_drpolicy_cli(switch_ctx=None):
         raise UnexpectedBehaviour(
             f"DRPolicy is not in succeeded or validated state: {status}"
         )
+
+
+def disable_dr_from_app(secondary_cluster_name):
+    old_ctx = config.cur_index
+    config.switch_acm_ctx()
+
+    # get all placement and replace value with surviving cluster
+    config.switch_acm_ctx()
```

Comment:
switching context twice. need to remove one.

Reply:
Done.removed